Admin/Configuration/AdminConfiguration.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+namespace LAG\AdminBundle\Admin\Configuration;
+
+use Doctrine\ORM\Mapping\ClassMetadata;
+
+/**
+ * Ease Admin configuration manipulation
+ */
+class AdminConfiguration
+{
+    /**
+     * @var string
+     */
+    protected $controllerName;
+
+    /**
+     * @var string
+     */
+    protected $entityName;
+
+    /**
+     * @var string
+     */
+    protected $manager;
+
+    /**
+     * @var string
+     */
+    protected $formType;
+
+    /**
+     * @var array
+     */
+    protected $actions = [];
+
+    /**
+     * @var int
+     */
+    protected $maxPerPage = 25;
+
+    /**
+     * @var string
+     */
+    protected $routingNamePattern;
+
+    /**
+     * @var string
+     */
+    protected $routingUrlPattern;
+
+    /**
+     * @var array
+     */
+    protected $adminConfiguration;
+
+    /**
+     * @var ClassMetadata
+     */
+    protected $metadata;
+
+    /**
+     * AdminConfiguration constructor.
+     *
+     * @param array $adminConfiguration
+     * @param ClassMetadata|null $metadata
+     */
+    public function __construct(array $adminConfiguration, ClassMetadata $metadata = null)
```

Comment:
Redundant naming for property adminConfiguration of the class AdminConfiguration.